youtube_dl/extractor/pluralsight.py

```diff
@@ -215,7 +219,7 @@ def _real_extract(self, url):
         for module in course_data:
             for clip in module.get('clips', []):
                 player_parameters = clip.get('playerParameters')
-                if not player_parameters:
+                if not player_parameters or clip.get('userMayViewClip') is False:
```

Comment:
Neither do I. But afair when I had one, `userMayViewClip` caused some problems by not always correlating with videos that can't actually be downloaded.